src/containers/CheckoutPage.less

```diff
@@ -0,0 +1,364 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+ @import './../style/common.less';
+ 
+.checkout-container {
+  position: relative;
+
+  .checkout-container-inner {
+    margin-left: -15px;
+    margin-right: -15px;
+
+    .view-title {
+      color: #000000;
+      text-transform: capitalize;
+
+      @media screen and (max-width: @mobileWidth - 1px) {
+        text-align: center;
+      }
+    }
+
+    .checkout-new-address-btn,
+    .checkout-edit-address-btn,
+    .checkout-delete-address-btn,
+    .checkout-new-payment-btn
+    {
+      border-radius: 4;
```

Comment:
You probably want to say 4px. Just 4 is huge. If you didn't notice huge border radiuses it is probably not applied and you can get rid of it. Otherwise convert to pixels.

Reply:
Chrome reads this as an invalid value and chooses to inherit. style.less is defining this at 4px so I will remove this one